packages/spark-core/components/_alerts.scss

```diff
@@ -0,0 +1,64 @@
+// ==================================================================
+// Alerts
+// ==================================================================
+.sprk-c-Alert {
+  background-color: $gray-light;
+  border-radius: $corners;
+}
+
+.sprk-c-Alert,
+.sprk-c-Alert__icon {
+  display: flex;
+}
+
+.sprk-c-Alert__icon,
+.sprk-c-Alert__text {
+  padding: $space-inset-m;
+}
+
+.sprk-c-Alert__text {
+  align-self: center;
+  flex: 1;
```

Comment:
This says for the text to take up the available space after the two icon areas are factored in.